### PR TITLE
fixes some nulling

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Fixes an error with gereration with card carp cards
Fixes an error with allowing you to null cells/items when crafting items using storage items or other tools/devices

### CODE REVIEW

File: code/datums/craft/step.dm [hunk 1]
```diff
@@ -133,6 +133,10 @@
 			to_chat(user, SPAN_WARNING("Wrong item!"))
 			building = FALSE
 			return
+		if(I.contents.len > 0)
+			to_chat(user, SPAN_WARNING("Their is something inside [I] that must be removed before crafting this!"))
```
Comment (ThePainkiller) [inline code/datums/craft/step.dm:137]:
```suggestion
			to_chat(user, SPAN_WARNING("There is something inside \the [I] that must be removed before using it to craft!"))
```

File: modular_liberty/cardgame_2/cardgame_2.dm [hunk 2]
```diff
@@ -487,78 +487,52 @@
...
-		if(prob(15) && over_draw.len) //Just goodluck
-			added_descs += ", [pick_n_take(over_draw)]"
+	if(prob(15) && over_draw.len) //Just goodluck
+		descadd += ", [pick(over_draw)]"
 
-	director_desc = "[director_desc] [base_descs] [added_descs]"
+	desc += descadd
 
+	if(new_name)
+		name = "Redink [new_name]"
```
Comment (ThePainkiller) [inline modular_liberty/cardgame_2/cardgame_2.dm:538]: What's "Redink"...?

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: modular_liberty/cardgame_2/cardgame_2.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply modular_liberty/cardgame_2/cardgame_2.dm:538]: Its the opposed to blackink and blueink is used already

File: code/datums/craft/step.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/datums/craft/step.dm:137]: Their not there

Comment (ThePainkiller) [reply code/datums/craft/step.dm]: This is...not gramatically correct Trilby...
"there is" = indicates something exists in that location
"their" = indicates that the object in question is possessed by someone you are referring to

**Pull request merged**